### PR TITLE
Feature 70 coach download mandate

### PR DESCRIPTION
**Pull request by sophiemoustard**

### CODE REVIEW

File: models/Customer.js [hunk 1]
```diff
@@ -41,7 +41,13 @@ const CustomerSchema = mongoose.Schema({
   payment: {
     bankAccountOwner: String,
     iban: String,
-    bic: String
+    bic: String,
+    sepa: [{
+      rum: String,
+      evenSignId: String,
```
Comment (jctrebalag) [inline models/Customer.js]: everSignId :)

File: models/Customer.js [hunk 2]
```diff
@@ -41,7 +41,13 @@ const CustomerSchema = mongoose.Schema({
...
+      drivId: String,
```
Comment (hofman-p) [inline models/Customer.js]: "drivId" -> "driveId" ^^

File: models/Customer.js (hunk 1 above)
Comment (sophiemoustard) [reply models/Customer.js]: Done

File: controllers/customerController.js [hunk 3]
```diff
@@ -100,140 +100,140 @@ const update = async (req) => {
...
-          subscriptions: { $exists: true },
-        }, 
-        { 'identity.firstname': 1, 'identity.lastname': 1, subscriptions: 1 },
-        { autopopulate: false }
-      ).lean();
-
-      if (!customer) {
-        return Boom.notFound(translate[language].customerSubscriptionsNotFound);
-      }
+const getSubcriptions = async (req) => {
```
Comment (hofman-p) [inline controllers/customerController.js]: "getSubcriptions" -> "getSubscriptions"

File: models/Customer.js (hunk 2 above)
Comment (sophiemoustard) [reply models/Customer.js]: Done

File: controllers/customerController.js (hunk 3 above)
Comment (sophiemoustard) [reply controllers/customerController.js]: done

**Pull request merged**